recipes/icc/all/test_package/CMakeLists.txt

```diff
@@ -0,0 +1,17 @@
+cmake_minimum_required(VERSION 3.17)
+project(PackageTest)
+
+set(CMAKE_CXX_STANDARD 11)
+
+include(${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
+conan_basic_setup()
+
+find_package(icc REQUIRED)
+
+add_executable(example example.cpp)
+target_include_directories(example PUBLIC ${CONAN_INCLUDE_DIRS_ICC})
+if (UNIX)
+    target_link_libraries(example icc::icc pthread)
+elseif(WIN32)
+    target_link_libraries(example icc::icc ws2_32 wsock32)
```

Comment:
these system libs need to be moved to the recipe like you had done for android

Reply:
Why ??
It is just test_package dependency ... Who cares ?

Reply:
Got it !!
Done